src/main.py

```diff
@@ -1,11 +1,29 @@
 import sys
+import os
+from Board import initBoard, getBoardFromConfig, displayBoard, shoot
+from constants import UNICODE_FOR_A_CHAR
+from Game import switchPlayer, initGame
+from utils import isValid
+from Config import getPlayersConfig
+import ui
 
-from Board import initBoard, getBoardFromConfig, displayBoard
+config1, config2 = getPlayersConfig()
 
-playerConfig = sys.argv[1].split("=")[1]
+boards, currentPlayer = initGame(config1, config2, 10)
+hit = 0
 
-with open("./config/" + playerConfig, 'r') as f:
-    config = f.read().splitlines()
+while True:
+    # Display current player board
+    ui.displayPlayerBoard(boards, currentPlayer, hit)
+    
+    # Ask for a coordinate
+    coord = input("Player " + str(currentPlayer) + ", where do you want to fire ?")
```

Comment:
```suggestion
    coord = input("Player " + str(currentPlayer) + ", where do you want to fire ? ")
```

To avoid  `where do you want to fire?B1`